### PR TITLE
Add instrumentations for python, ruby, JS to registry.

### PR DESCRIPTION
**Pull request by svrnm**
Some more missing registry entries.

### CODE REVIEW

File: content/en/registry/instrumentation-js-fs.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: OpenTelemetry `fs` Instrumentation for Node.js
```
Comment (chalin) [inline content/en/registry/instrumentation-js-fs.md:2]: Maybe remove the markdown syntax here?

```suggestion
title: OpenTelemetry fs Instrumentation for Node.js
```

**Review by chalin — APPROVED** [review]
Minor comment, otherwise (RS)LGTM

File: content/en/registry/instrumentation-js-fs.md (hunk 1 above)
Comment (svrnm) [reply content/en/registry/instrumentation-js-fs.md:2]: I removed the markdown in the title/description everywhere because it's not parsed in the registry

Comment (cartermp) [reply content/en/registry/instrumentation-js-fs.md:2]: I'm a little confused here. Is the markdown intended to be removed or not? It's not removed in the preview of the site: https://deploy-preview-1930--opentelemetry.netlify.app/registry/?s=fs&component=&language=

**Review by cartermp — APPROVED** [review]

**Pull request merged**